code/modules/surgery/organs/organ_internal.dm

```diff
@@ -53,6 +73,14 @@
 	return
 
 /obj/item/organ/proc/on_life()
+	var/mob/living/carbon/C = owner
+	//if we start to fail, cap our damage and fail the organ
+	if(damage > maxHealth)
+		failing = TRUE
+		damage = maxHealth
+	//repair organ damage if the organ is not failing
```

Comment:
not all organs should be autohealing (eyes for example) so this will need to be addressed in the subtypes OR given a var like `passively_heals` and this block below checks for it if you need subtypes to run inher too.

Reply:
There is a var for that, Cobby, it's healing_factor, commented as such

Reply:
I'll change the comment on it to be more clear, since it doesn't really heal that fraction per tick but when on_life() is called.

Reply:
don't worry about it I'm trying the new view files thing but the problem with closing the file is I can't easily go back and look.

just say ur r****** and resolve lmao